### DISCUSSION TITLE
SSL not working

### DISCUSSION THREAD

**Discussion by jordan-adelle** [lavalink-devs/Lavalink#753] [Q&A] [answered]
Good evening,
We can't put a domain on lavalink, I already tried to change the host to put a local configuration etc... it's just not working.

Screen application.yml
![image](https://user-images.githubusercontent.com/52496524/192167902-d6223819-facb-4223-b012-8aa098eca491.png)

Screen NGINX
![image](https://user-images.githubusercontent.com/52496524/192167920-0e58cfa8-751c-45f2-b828-207db50639e0.png)

**Comment by jordan-adelle** [comment]
I already tried to change the host to lavalink.polario.xyz

**Comment by topi314** [comment, marked as answer]
it seems like you're missing the websocket upgrade header
see: https://www.nginx.com/blog/websocket-nginx/
